rxjs-1/package.json

```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "csvsync": "^1.1.0",
+    "easy-table": "^1.1.1",
```

Comment:
I suggest you should commit that one on the main repo to avoid a new major version being released and causing mayhem for the next candidates